js/parallax.js

```diff
@@ -110,16 +110,18 @@ const asteroidProto = {
                 dy *= -1;
             }
             
-            this.setPosition(x, y, dx, dy);
+            this.setPosition(x, y, dx, dy, randomFloat(-2.5, 5));
         }
         setTimeout(() => entityHandler.registerEntity(this), 250);
         parallax.appendChild(this.element);
     },
-    setPosition(x, y, dx, dy) {
+    setPosition(x, y, dx, dy, rot) {
         this.x = x;
         this.y = y;
         this.dx = dx;
         this.dy = dy;
+        this.rotPerFrame = rot;

```

Comment:
`rotationPerFrame` or perhaps `dRotation`

Reply:
imho `rotationPerFrame` - characters are pretty cheap nowadays and that's nice and clear :-)